ibis/backends/tests/test_dot_sql.py

```diff
@@ -306,3 +306,12 @@ def test_order_by_no_projection(backend):
 
     result = con.sql(ibis.to_sql(expr)).execute().name.iloc[:2]
     assert set(result) == {"Ross, Jerry L.", "Chang-Diaz, Franklin R."}
+
+
+@dot_sql_notimpl
+@dot_sql_notyet
+@dot_sql_never
+@pytest.mark.notyet(["polars"], raises=PolarsComputeError)
+def test_dot_sql_limit(con):
+    expr = con.sql("select 'abc' ts").limit(1)
```

Comment:
Can you select from a table with more than 1 rows?

Reply:
Is one row not sufficient? I don't think the issue is not related to the number of rows.

Reply:
Added more rows to the test.

Reply:
Actually, I'm going to keep the original test. Testing multiple rows adds unnecessary complexity.

Reply:
I basically have to solve the dialect problem, whereas `SELECT 'abc' ts` is valid in every SQL dialect we support.